quipucords/api/source/model.py

```diff
@@ -50,6 +50,7 @@ class Source(BaseModel):
     hosts = models.JSONField(unique=False, null=False, default=list)
     exclude_hosts = models.JSONField(unique=False, null=True)
 
+    # TODO Evaluate if we even need to keep this relationship.
```

Comment:
I was considering adding something similar for v2 API, but linking to `ScanJob` instead. However, that should be a queryset annotation IMO, not an actual column.